src/shared/api/socket-api.ts

```
@@ -0,0 +1,26 @@
+class SocketApi {
+  static socket: Socket | null = null
+
+  static createConnection() {
+    const accessToken = localStorage.getItem(ACCESS_TOKEN)?.replace(/"/g, '') || ''
+    const queryParams = {
+      query: {
+        accessToken: accessToken,
+      },
+    }
+
+    this.socket = io('https://inctagram.work', queryParams)
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)